data-sample/semopenalex-data-sample.trig

```diff
@@ -0,0 +1,2494 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix ns1: <https://semopenalex.org/property/> .
+@prefix ns2: <http://purl.org/spar/datacite/> .
+@prefix ns3: <http://purl.org/spar/cito/> .
+@prefix ns4: <http://purl.org/spar/fabio/> .
+@prefix ns5: <http://purl.org/spar/bido/> .
+@prefix ns6: <https://dbpedia.org/ontology/> .
+@prefix ns7: <http://prismstandard.org/namespaces/basic/2.0/> .
+@prefix ns8: <https://dbpedia.org/property/> .
+@prefix ns9: <http://www.geonames.org/ontology#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ns1:context {
```

Comment:
should we use `https://semopenalex.org/sample-dataset/context` namespace?

Reply:
Yes that's a good point. So change line 18 to: <https://semopenalex.org/sample-dataset/context> {        
?